core/metacling/src/TClingValue.cxx

```diff
@@ -44,6 +44,25 @@ TClingValue& TClingValue::operator=(TClingValue &Other) {
    return *this;
 }
 
+std::pair<std::string, std::string> TClingValue::ToTypeAndValueString() const {
+  std::string output = ToString();
+  int pos, bracket = 0, str_size = output.size() - 1;
+  for (pos = 0; pos <= str_size; pos++) {
+    if (output[pos] == '(')
+      bracket++;
+    else if (output[pos] == ')') {
+      bracket--;
+      if (!bracket)
+        break;
```

Comment:
Why don't you return the `pair` here, and return "error case" outside the for loop?

That will allow you to make `pos`, `str_size` proper for loop variables (i.e. scoped inside the loop)